src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -787,7 +788,7 @@ TTL74273 = 74273 : bascule octale en D avec bascule transparente
 TTL74283 = 74283 : Additionneur complet binaire 4 bits
 TTL7430 = 7430 : porte NAND simple à 8 entrées
 TTL7432 = 7432 : quad 2 entrées OU porte OU
-TTL7434 = 74157 : quadruple 2-lignes to 1 ligne sélecteur
+TTL7434 = 7434 : tampon hexagonal
```

Comment:
According to [Wikipedia](https://fr.wikipedia.org/wiki/Liste_des_circuits_intégrés_de_la_série_7400), I suggest to use "six tampons non-inverseurs".